userguide/tutorials/getting_started.adoc

```diff
@@ -136,47 +136,58 @@ You can also go to http://<IP>:8080/api.html to explore the KillBill APIs.
 
 === Tomcat
 
-Users familiar with Java technologies can also install Kill Bill directly in their Web container of choice, such as Tomcat. We recommend using Tomcat version 8.5.
+Users familiar with Java technologies can also install Kill Bill directly in the Web container of their choice, such as Tomcat. We recommend using Tomcat version 8.5.
 
-If you don't have it yet, you can download the Tomcat container http://tomcat.apache.org/download-80.cgi[here] (get the Core binary distribution). The documentation is available http://tomcat.apache.org/tomcat-8.0-doc/index.html[here].
+==== Tomcat Installation
 
-In the rest of this guide, we will assume you have installed Tomcat in a directory called *CATALINA_HOME*.
+1. Download the Tomcat container from http://tomcat.apache.org/download-80.cgi[here] (get the Core binary distribution). 
 
-Notes:
+1. Install Tomcat as per the instructions specified http://tomcat.apache.org/tomcat-8.0-doc/index.html[here].
+
+1. The rest of the guide assumes that you have installed Tomcat in a directory called *CATALINA_HOME*.
```

Comment:
I believe you need to either use `.` or specify the numbers in order (so `2.` here): https://asciidoctor.org/docs/asciidoc-writers-guide/#ordering-the-things

Reply:
Actually, asciidoc will indeed do the right thing. Although, it's a bit confusing. We've decided to stick to `.` instead.

Reply:
Changed to use `.`